core/src/apps/webauthn/add_resident_credential.py

```diff
@@ -41,15 +41,15 @@ async def add_resident_credential(
     try:
         cred = Fido2Credential.from_cred_id(bytes(msg.credential_id), None)
     except Exception:
-        text = Text("Import credential", ui.ICON_WRONG, ui.RED)
-        text.normal(
-            "The credential you are",
-            "trying to import does",
-            "not belong to this",
-            "authenticator.",
+        await show_error_and_raise(
+            ctx,
+            "warning_credential",
+            header="Import credential",
+            button="Close",
+            content="The credential you are trying to import does\nnot belong to this authenticator.",
+            red=True,
+            exc=wire.ActionCancelled,
```

Comment:
this is the default, no?

Reply:
Fixed in fef81373696d5dc2929b40dba0343afe855c9a7f.